src/workflows/steps.js

```diff
@@ -468,3 +470,152 @@ export function setPromote( state ) {
 export function verifyConflictResolution() {
 	return git.checkConflictMarkers();
 }
+
+export function getPackageScope( state ) {
+	if ( state.qa && typeof state.qa === "boolean" ) {
+		state.qa = "@lk";
+	} else if ( state.qa && state.qa.charAt( 0 ) !== "@" ) {
+		state.qa = `@${ state.qa }`;
+	}
+
+	return Promise.resolve();
+}
+
+export function getScopedRepos( state ) {
+	const { configPath, qa: scope } = state;
+	let content = {};
+	try {
+		content = util.readJSONFile( configPath );
```

Comment:
you could probably move the `let` on line 486 to this line since it's not needed outside the try/catch block